packages/editor/src/client/app.tsx

```diff
@@ -121,5 +125,26 @@ export function App() {
 
   return null
 }
+export function App() {
+  const {current} = useRoute()
 
+  const {i18n} = useTranslation()
+  // eslint-disable-next-line @typescript-eslint/ban-types
```

Comment:
you can get rid of the eslint comment by using `Record<string, unknown>` on line 133 instead of `object`.

Reply:
This one is not giving any errors, so I just removed the comment